enhancements/olm/single-ownnamespace-enhancement-proposal.md

```diff
@@ -0,0 +1,524 @@
+
+This enhancement provides **limited compatibility support for existing registry+v1 bundle content only**. The `watchNamespace` configuration is treated as opaque bundle-provided configuration used solely for rendering registry+v1 bundles into appropriate manifests. This is not a multi-tenancy feature but rather a bundle format compatibility layer to support migration from OLM v0 to OLM v1.
+
+Future OLM v1 bundle formats will not include these legacy install mode concepts, as OLM v1 will not have first-class features or opinions on RBAC management in next-generation bundle formats.
+
+## Motivation
+
+The registry+v1 bundle format (used by OLM v0) includes install mode declarations (AllNamespaces, MultiNamespace, SingleNamespace, OwnNamespace) that affect how bundles are rendered into Kubernetes manifests. OLM v1 deliberately simplified operator installation by focusing on AllNamespaces mode to avoid complexity, but there exists significant existing registry+v1 bundle content in catalogs that only declares support for Single and OwnNamespace install modes.
+
+### User Stories
+
+#### Story 1: Legacy Operator Migration
+As a cluster administrator migrating from OLM v0 to OLM v1, I want to install operators that only support SingleNamespace or OwnNamespace install modes so that I can continue using existing operator content without requiring operator authors to modify their bundles.
+
+#### Story 2: Operator Author Requirements
+As an operator developer, I have existing registry+v1 bundles that only support Single or OwnNamespace install modes, and I want my customers to be able to deploy these operators in OpenShift with OLM v1 so that the bundle content can be properly rendered and installed without requiring me to modify my existing bundle format during the migration to OLM v1.
+
+### Goals
+
+- Enable installation of registry+v1 bundles that only support SingleNamespace or OwnNamespace install modes
+- Provide a configuration mechanism through the ClusterExtension API to pass opaque watchNamespace configuration for bundle rendering
+- Maintain backward compatibility with existing registry+v1 bundle content from OLM v0 catalogs
+- Generate appropriate RBAC resources scoped to the target namespaces for registry+v1 bundles only
+- Support seamless migration path for customers moving from OLM v0 to OLM v1
+
+#### Future Goals
+
+- Expand the configuration surface for registry+v1 bundles to support OLMv0 SubscriptionConfig-type behavior, enabling broader operator configuration compatibility during the migration from OLMv0 to OLMv1 
+
+### Non-Goals
+
+- Re-introducing multi-tenancy features or supporting multiple installations of the same operator
+- Supporting MultiNamespace install mode (watching multiple namespaces)
+- Modifying the fundamental OLM v1 architecture or adding complex multi-tenancy logic
+- Supporting install mode switching after initial installation as a first-class feature
```

Comment:
Maybe we could remove this from dot point from here and point out (with more words) that changing the watchNamespace may work differently for different bundles. I.e. ClusterExtension allows you to change it, it computes the correct resource set, it pivots towards that resource set, but whether that ends up in a working application if application dependent...?